packages/salesforcedx-vscode-lightning/src/index.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as languageServer from '@salesforce/salesforcedx-slds-linter/out/src/client';
+import * as vscode from 'vscode';
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log('SFDX SLDS Linter Extension Activated');
+  vscode.workspace.findFiles('**/staticresources/*.resource').then(
+    // all good
+    (result: vscode.Uri[]) => {
+      for (let i = 0; i < result.length; i++) {
```

Comment:
Cosmetic: Is there a reason we use `for(let i = 0...)` instead of the for(const a of blah)?